scripts/install.sh

```diff
@@ -7,6 +7,40 @@ SYSTEMD_PATH_FILE="/etc/systemd/system/cs-custom-bouncer.service"
 API_KEY=""
 BINARY_PATH=""
 
+usage() {
+      echo "Usage: ./install.sh [options]"
+      echo "    -h|--help                              Display this help message."
+      echo "    -b|--binary <path>                     Specify the binary path"
+
+      exit 0
+}
+
+while [[ $# -gt 0 ]]
+do
+    key="${1}"
+    case ${key} in
+    -b|--binary)
+        if ! [ -f "${2}" ]; then
+            echo "${key} need a path"
+            usage
+            exit 1
+        fi
+        BINARY_PATH="$2"
+        shift # past argument
```

Comment:
Doing some validation of whether the path is absolute would be helpful. 
```suggestion
        shift # past argument
        if ! [[ $BINARY_PATH == /* ]]; then
            echo "binary path must be absolute"
            exit 1
        fi
```

Reply:
I think changing doc ( https://github.com/crowdsecurity/cs-custom-bouncer/pull/10/files#diff-e076e43cb817ecc8b3c2ed18cd58baa50544ecc66293ed6e61129c43ca6a09ccR24 ) to `need an absolute path` can be a good idea too .

Reply:
Let's support relative path too: 
```suggestion
        shift # past argument
        BINARY_PATH=$(readlink -f $BINARY_PATH)
```
Just adding this one line does the trick :) 
Credit to @AlteredCoder